ui/admin/tests/acceptance/targets/create-test.js

```diff
@@ -111,7 +102,7 @@ module('Acceptance | targets | create', function (hooks) {
   });
 
   test('can create type `ssh` when `target-worker-filters-v2` is enabled', async function (assert) {
-    featuresService.enable('target-worker-filters-v2');
+    enableFeature('target-worker-filters-v2');
```

Comment:
None blocking comment: I am wondering about the context of enableFeature.

Enabling a specific feature in a test , will be persisted for the next test case? or each test case starts with a default fresh state of feature flags?

In other words, If we enable `target-worker-filters-v2` in this test, in the next test within the same describe block, will `target-worker-filters-v2` be enabled or disable?

Reply:
I have not run into issues with enabling a feature in one test case and it still being enabled in another test case, even within the same test module. I think if this was an issue, `yarn test` would likely never be successful as it randomizes our tests to ensure stuff like this doesn't happen.

Reply:
@cameronperera you are right, if sharing scope out of the `test()` function our test suite would fail. But I was wondering if we explicitly did the sanity check with this 👍